### ISSUE TITLE
Can non angular applications be mocked using protractor?

### ISSUE THREAD

**Issue by abhinaba1080** [angular/protractor#5402]
Hi ,

I am using browser.ignoreSynchronization=true for my non angular application. So, protractor loose track of the http calls in backend. So, while trying any of the mock libraries available for protractor it is not working. 

Is there a way to mock non angular applications with protractor?

**Comment by Fuun347** [comment]
By what I've read online browser.ignoreSynchronization is deprecated. The recommended way is using browser.waitForAngularEnabled(false)

**Comment by abhinaba1080** [comment]
@Fuun347 

Yup even if you write so, protractor still loose track the http asynchronous calls. In this circumstances, what is the way to setup mocks for my protractor tests?

**Comment by Fuun347** [comment]
Not too sure myself as I've never had to work with mocks, but this issue here might have an answer for you https://github.com/angular/protractor/issues/125

**Comment by SinghRajKr** [comment]
Try following in your spec.js file 
Try setting false for non-angular app page
browser.waitForAngularEnabled(false);
Try setting true for angular app page
browser.waitForAngularEnabled(true);
